demo/src/components/Footer.astro

```diff
@@ -0,0 +1,9 @@
+---
+import { t } from "i18n:astro";
+
+---
+<footer class="bg-gray-300 p-4 sticky bottom-0 flex justify-center items-center text-center" style="position:absolute;width:100%;padding-bottom:2rem;">
```

Comment:
Can we use only tailwind and not inline styles?

Reply:
oops that was supposed to be temporary while i figured another way to do it :P
in my todo list :)